Jenkinsfile

```diff
@@ -190,16 +190,13 @@ pipeline {
                                     githubNotify(credentialsId: 'bbrks_uberjenkins_sg_access_token', context: 'sgw-pipeline-errcheck', description: 'Running', status: 'PENDING')
                                     sh "errcheck ${SGW_REPO}/... | tee errcheck.out"
                                     sh "test -z \"\$(cat errcheck.out)\""
-                                    githubNotify(credentialsId: 'bbrks_uberjenkins_sg_access_token', context: 'sgw-pipeline-errcheck', description: 'OK', status: 'SUCCESS')
                                 } catch (Exception e) {
                                     sh "wc -l < errcheck.out | awk '{printf \$1}' > errcheck.count"
                                     script {
                                         env.ERRCHECK_COUNT = readFile 'errcheck.count'
                                     }
-                                    // TODO: Remove in CBG-492
-                                    githubNotify(credentialsId: 'bbrks_uberjenkins_sg_access_token', context: 'sgw-pipeline-errcheck', description: "found "+env.ERRCHECK_COUNT+" unhandled errors", status: 'SUCCESS')
-//                                     githubNotify(credentialsId: 'bbrks_uberjenkins_sg_access_token', context: 'sgw-pipeline-errcheck', description: "found "+env.ERRCHECK_COUNT+" unhandled errors", status: 'FAILURE')
-//                                     unstable("errcheck failed")
+                                } finally {
+                                    githubNotify(credentialsId: 'bbrks_uberjenkins_sg_access_token', context: 'sgw-pipeline-errcheck', description: 'OK', status: 'SUCCESS')
```

Comment:
We're always reporting success here now with the new `finally` block, even when errcheck fails.

I think we should retain the existing 'SUCCESS' path above in the `try`, and report a 'FAILURE' status in the catch like the commented code was doing, and not forgetting `unstable` to mark the job as having failures.

Reply:
Makes sense. Fixed in latest revision.